sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py

```diff
@@ -48,12 +49,15 @@
 NEW_TYPES_OUTPUT_SCHEMA = (
     '{"fields": [{"name": "bytes","type": "BYTES"},'
     '{"name": "date","type": "DATE"},{"name": "time","type": "TIME"}]}')
-NEW_TYPES_OUTPUT_VERIFY_QUERY = ('SELECT date FROM `%s`;')
-# There are problems with query time and bytes with current version of bigquery.
+NEW_TYPES_OUTPUT_VERIFY_QUERY = ('SELECT bytes, date, time FROM `%s`;')
 NEW_TYPES_OUTPUT_EXPECTED = [
-    (datetime.date(2000, 1, 1),),
-    (datetime.date(2011, 1, 1),),
-    (datetime.date(3000, 12, 31),)]
+    (b'xyw=', datetime.date(2011, 1, 1), datetime.time(23, 59, 59, 999999),),
+    (b'abc=', datetime.date(2000, 1, 1), datetime.time(0, 0),),
+    (b'dec=', datetime.date(3000, 12, 31),
+     datetime.time(23, 59, 59, 999999),),
+    (b'\xab\xac\xad', datetime.date(2000, 1, 1), datetime.time(0, 0),)]
+NEW_TYPES_OUTPUT_EXPECTED = [(base64.b64encode(row[0]).decode('utf-8'),)
```

Comment:
`base64.b64encode` here makes the test work around the limitation in IO. I think we need to understand first what is the intended behavior of the IO, we can continue the conversation on the associated Jira.

Reply:
This was my mistake. The test now succeeds without encoding the `NEW_TYPES_OUTPUT_EXPECTED`

What the test does:
- test setup: write data to bq using bq client using base-64
- test: read this data and write it to a new table (reads data in base-64 and writes it in base-64,  this is why the test can succeed)
- matcher: queries data using bq client, this returns bytes and is than matched to the bytes in `NEW_TYPES_OUTPUT_EXPECTED`